### PR TITLE
[FMT] format example/03_connectivity

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to https://github.com/nilearn/nilearn/issues/2528

Changes proposed in this pull request:

- apply pre-commit hooks to example/03_connectivity

### Files in other PRs
- 'examples/03_connectivity/plot_group_level_connectivity.py',

### CODE REVIEW

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 1]
```diff
@@ -110,9 +114,11 @@
 # partial correlations.
 
 from nilearn.connectome import ConnectivityMeasure
-connectivity_measure = ConnectivityMeasure(kind='partial correlation')
-partial_correlation_matrix = connectivity_measure.fit_transform(
-    [time_series])[0]
+
+connectivity_measure = ConnectivityMeasure(kind="partial correlation")
+partial_correlation_matrix = connectivity_measure.fit_transform([time_series])[
+    0
```
Comment (bthirion) [inline examples/03_connectivity/plot_sphere_based_connectome.py:120]: :-(

**Review by bthirion — COMMENTED** [review]
LGTM overall

File: examples/03_connectivity/plot_sphere_based_connectome.py (hunk 1 above)
Comment (Remi-Gau) [reply examples/03_connectivity/plot_sphere_based_connectome.py:120]: yup we will inevitably get those weird line splits, but that would be the case with any line length.

**Review by ymzayek — APPROVED** [review]
Thx @Remi-Gau the changes LGTM overall. The only thing is that the f-string updates seems not to be consistent

File: examples/03_connectivity/plot_atlas_comparison.py [hunk 2]
```diff
@@ -33,37 +33,48 @@
...
+print(
+    "Functional nifti images (4D, e.g., one subject) are located at : %r"
+    % data["func"][0]
+)
+print(
+    "Counfound csv files (of same subject) are located "
+    f"at : {data['confounds'][0]!r}"
+)
+
+from nilearn.connectome import ConnectivityMeasure
```
Comment (NicolasGensollen) [inline examples/03_connectivity/plot_atlas_comparison.py:55]: Should go below comment

File: examples/03_connectivity/plot_multi_subject_connectome.py [hunk 3]
```diff
@@ -26,59 +25,81 @@ def plot_matrices(cov, prec, title, labels):
...
 # print basic information on the dataset
-print('First subject functional nifti image (4D) is at: %s' %
-      rest_dataset.func[0])  # 4D data
+print(
+    f"First subject functional nifti image (4D) is at: {rest_dataset.func[0]}"
+)
 
 
+# A "memory" to avoid recomputation
+from joblib import Memory
```
Comment (NicolasGensollen) [inline examples/03_connectivity/plot_multi_subject_connectome.py:62]: I think we can remove the associated comment and move the import after the the "Extracting region signals" title below.

File: examples/03_connectivity/plot_multi_subject_connectome.py [hunk 4]
```diff
@@ -26,59 +25,81 @@ def plot_matrices(cov, prec, title, labels):
...
+
 ##############################################################################
 # Extracting region signals
 # --------------------------
 from nilearn.maskers import NiftiMapsMasker
 
-# A "memory" to avoid recomputation
-from joblib import Memory
-mem = Memory('nilearn_cache')
+mem = Memory("nilearn_cache")
```
Comment (NicolasGensollen) [inline examples/03_connectivity/plot_multi_subject_connectome.py:69]: Are we using `mem` somewhere ? It's not obvious to see where it is used by looking at the diff...

**Review by NicolasGensollen — APPROVED** [review]
Thanks @Remi-Gau !
LGTM with a few small suggestions.

File: examples/03_connectivity/plot_multi_subject_connectome.py (hunk 4 above)
Comment (Remi-Gau) [reply examples/03_connectivity/plot_multi_subject_connectome.py:69]: pretty sure flake8 would complain about some unused variable if we did not, but let me cross check

Comment (Remi-Gau) [reply examples/03_connectivity/plot_multi_subject_connectome.py]: actually I was mistaken flake8 won't say anything for scripts but [vulture](https://pypi.org/project/vulture/) did flag this as dead code along with a few other variables.

File: examples/03_connectivity/plot_multi_subject_connectome.py (hunk 3 above)
Comment (Remi-Gau) [reply examples/03_connectivity/plot_multi_subject_connectome.py]: Actually along with your other comment, it seems that this is not actually used anywhere so I removed it.

Unless I am missing something on how this caching is used.

File: examples/03_connectivity/plot_atlas_comparison.py (hunk 2 above)
Comment (Remi-Gau) [reply examples/03_connectivity/plot_atlas_comparison.py]: done

File: examples/03_connectivity/plot_atlas_comparison.py [hunk 5]
```diff
@@ -33,37 +33,48 @@
 from nilearn import datasets
 
 yeo = datasets.fetch_atlas_yeo_2011()
-print('Yeo atlas nifti image (3D) with 17 parcels and liberal mask is located '
-      'at: %s' % yeo['thick_17'])
+print(
+    "Yeo atlas nifti image (3D) with 17 parcels and liberal mask is located "
+    "at: %s" % yeo["thick_17"]
```
Comment (Remi-Gau) [inline examples/03_connectivity/plot_atlas_comparison.py:38]: TODO fix f string

File: examples/03_connectivity/plot_atlas_comparison.py [hunk 6]
```diff
@@ -33,37 +33,48 @@
...
 # --------------------
 data = datasets.fetch_development_fmri(n_subjects=10)
 
-print('Functional nifti images (4D, e.g., one subject) are located at : %r'
-      % data['func'][0])
-print('Counfound csv files (of same subject) are located at : %r'
-      % data['confounds'][0])
+print(
+    "Functional nifti images (4D, e.g., one subject) are located at : %r"
+    % data["func"][0]
```
Comment (Remi-Gau) [inline examples/03_connectivity/plot_atlas_comparison.py:48]: TODO fix f string

File: examples/03_connectivity/plot_seed_to_voxel_correlation.py [hunk 7]
```diff
@@ -157,33 +179,41 @@
...
 # --------------------------------------
 # Finally, we can Fisher-z transform the data to achieve a normal distribution.
 # The transformed array can now have values more extreme than +/- 1.
 seed_to_voxel_correlations_fisher_z = np.arctanh(seed_to_voxel_correlations)
-print("Seed-to-voxel correlation Fisher-z transformed: min = %.3f; max = %.3f"
-      % (seed_to_voxel_correlations_fisher_z.min(),
-         seed_to_voxel_correlations_fisher_z.max()
-         )
-      )
+print(
```
Comment (Remi-Gau) [inline examples/03_connectivity/plot_seed_to_voxel_correlation.py:203]: fix f strings

**Pull request merged**